features/step_definitions/common_stepdef.rb

```diff
@@ -158,14 +140,48 @@
   end
 end
 
-Then(/^I create a new procurement/) do
-  step 'I am a logged in user - buildings account'
-  step 'I am on your account page'
-  step 'I click on start a procurement'
-  step 'I am on "What happens next" page'
-  step 'I click on "Continue"'
-  step 'I am on "Contract name" page'
-  step 'I add contract name'
-  step 'I click on "Save and continue"'
-  step 'I am on "Requirements" page'
+And(/^I should see error message header "([^"]*)"$/) do |heading|
+  expect(common.header_two.text).to eq(heading)
+end
+
+And(/^I should see error message header "([^"]*)"$/) do |heading|
+  expect(common.header_two.text).to eq(heading)
+end
```

Comment:
This code is repeated on lines 143 - 146

Reply:
Removed repeated code